src/extensions/service-managers/civicrm-donotsms/react-component.js

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable react/no-multi-comp */
+/* eslint no-console: 0 */
+import PropTypes from "prop-types";
+import React from "react";
+import Form from "react-formal";
+import * as yup from "yup";
+import GSForm from "../../../components/forms/GSForm";
+import GSTextField from "../../../components/forms/GSTextField";
+import GSSubmitButton from "../../../components/forms/GSSubmitButton";
+
+export class OrgConfig extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+
+    const formSchema = yup.object({
+      savedText: yup
+        .string()
+        .nullable()
+        .max(64)
+    });
+    return (
+      <div>
+        THIS IS TEST_FAKE_DATA
```

Comment:
Is this here by accident? On purpose? Same question with the other FAKE_DATA further down